tests/cross/src/test/scala/tests/pc/CompletionMatchSuite.scala

```diff
@@ -103,6 +107,63 @@ object CompletionMatchSuite extends BaseCompletionSuite {
     filter = _.contains("exhaustive")
   )
 
+  checkEdit(
+    "exhaustive-sorting",
+    """package sort
+      |sealed abstract class TestTree
+      |case class Branch1(t1: TestTree) extends TestTree
+      |case class Leaf(v: Int) extends TestTree
+      |case class Branch2(t1: TestTree, t2: TestTree) extends TestTree
+      |object App {
+      |  null.asInstanceOf[TestTree] matc@@
+      |}
+      |""".stripMargin,
+    """|package sort
+       |sealed abstract class TestTree
+       |case class Branch1(t1: TestTree) extends TestTree
+       |case class Leaf(v: Int) extends TestTree
+       |case class Branch2(t1: TestTree, t2: TestTree) extends TestTree
+       |object App {
+       |  null.asInstanceOf[TestTree] match {
+       |\tcase Branch1(t1) => $0
+       |\tcase Leaf(v) =>
+       |\tcase Branch2(t1, t2) =>
+       |}
+       |}
+       |""".stripMargin,
+    filter = _.contains("exhaustive")
+  )
+
+  if (!isScala211) {
+    checkEdit(
+      "exhaustive-sorting-scalalib",
+      """package sort
+        |object App {
+        |  Option(1) matc@@
+        |}
+        |""".stripMargin,
+      if (!isScala211)
+        """package sort
+          |object App {
+          |  Option(1) match {
+          |\tcase Some(value) => $0
+          |\tcase None =>
+          |}
+          |}
+          |""".stripMargin
+      else
```

Comment:
I don't think we need the else here? `!isScala211` is already tested at the top

Reply:
Oops, we don't need it. I meant like this https://github.com/scalameta/metals/pull/1174/commits/0d5f96fb977b6c73c44b8b5c061480393ecb1c82, thank you for pointing it out :)